bin/lambda-local

```diff
@@ -52,6 +52,8 @@
 		awsRequestId: _context.createInvokeId,
 		timeoutMs: timeoutMs
 	});
+    // export the LAMBDA_TASK_ROOT enviroment variable
+    process.env['LAMBDA_TASK_ROOT'] = process.cwd();
```

Comment:
> As far as I know, LAMBDA_TASK_ROOT is the directory where the zip is exracted

Yes that seems right. Still, neither `__dirname` nor `process.cwd()` satisfies that.. So I wonder what it should be. Maybe the value of `-l` (the path to the lambda function)? 
I will look into it and will revise it when I find a better solution; 
Meanwhile, having a not-so-accurate `LAMBDA_TASK_ROOT` is better than having none, so I will go ahead and merge this!